.github/workflows/tests.yml

```diff
@@ -21,10 +24,23 @@ jobs:
           python --version
           python -m pip install --upgrade pip
           python -m pip install --upgrade flit
```

Comment:
```suggestion
```

Since you're no longer invoking the Flit frontend but rather the `flit-core` backend, which gets installed in an isolated environment by `pip install -e .` and this version is ignored anyway, this line can be omitted.